Gemfile

```diff
@@ -9,7 +9,7 @@ end
 
 group :test, :development do
   gem 'rake', '>= 0.7.3'
-  gem 'rspec', '>= 2.9.0'
+  gem 'rspec', '~> 2.14.0'
```

Comment:
Doh..I just realized I must have already had a fork of this repo that was out of date. I'm merging upstream master to catch up. Sorry about that :smile: